content/en/docs/tutorials/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Tutorials
-description: A set of guides that walks you through using the different capabilities and features of Notary
+description: A set of guides that walks you through using the different capabilities and features of The Notary Project
```

Comment:
Suggest using Notation if this tutorials are for Notation

Reply:
holding off until #312 is resolved